platform-ui-skin/src/main/webapp/skin/less/social/portlets/profileSettings/Style.less

```diff
@@ -0,0 +1,104 @@
+/*
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+@import "../../../variables.less";
+@import "../../../mixins.less";
+#profileSettings {
+    background: @baseBackground!important;
+    #settingsHeader {
+        background: @baseBackground;
+       #filterSettingsSelect {
+          height: 40px;
+          width: 150px;
+        }
+    }
+
+    #profileSettingFormDrawer{
+        .selectLanguageFilter {
+          height: 40px!important;
```

Comment:
Please reuse the same component definition as other UIs for select boxes to avoid having specific UI for this portlet. See an example here:

https://github.com/Meeds-io/social/blob/4d7b3859e722921f757fb94c661c6de938867bd1/webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamFilter.vue#L19-L29